frontend/src/components/organization/OrganizationPreviewBody.js

```diff
@@ -0,0 +1,92 @@
+import { Box, CardContent, Tooltip, Typography } from "@material-ui/core";
+import React, { useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import PlaceIcon from "@material-ui/icons/Place";
+
+const useStyles = makeStyles((theme) => {
+  return {
+    bold: {
+      fontWeight: "bold",
+    },
+    locationName: {
+      fontWeight: "bold",
```

Comment:
I tested it locally and I think making the location bold highlights it a bit too much. I think semi-bold is the better choice here.
```suggestion
      fontWeight: 600,
```